CHANGELOG.md

```diff
@@ -8,20 +8,22 @@ and this project adheres to
 
 ## [Unreleased]
 
-### Changed
-
-- Activate pagination by default on all endpoints (20 items per page)
-- Use user fullname instead of username in order confirmation email
-- Rename certificate field into certificate_definition for the ProductSerializer
-- Improve certificate serializer
-
 ## Added
 
 - CRUD admin API for organizations, products, course runs,
   certificate definitions, courses and retrieve enabled languages
 - Allow on-demand page size on the order and enrollment endpoints
 - Add yarn cli to generate joanie api client in TypeScript
 - Display course runs into the admin course change view
+- Automatically allocate organization equitably on order creation according
+  their active order count per organization for the couple product, course
```

Comment:
```suggestion
- Automatically allocate organization equitably on order creation according to
  their active order count per organization for the product/course couple
```

Reply:
I'm wondering if it should only be "active" count or also orders pending payment otherwise you may assign several orderrs to the same organization in the meantime the paymnet is being made? I know it will "equalize" later so no big deal but still.